src/webgl/shaders/filters/dilate.frag

```diff
@@ -0,0 +1,38 @@
+// Increase the bright areas in an image
```

Comment:
Same for erode, but it will be a min instead of max

Reply:
`max()` seems to choose that greatest of each component, compared to the `luma()` way of choosing the greatest overall vector. Example:
```
a = (1, 1, 0)
b = (0, 0, 1)
max(a,b) = (1, 1, 1)  # instead of (1, 1, 0)
```

Not sure if that's more desired or not. The results look similar though

Reply:
I think either way is fine, if the luma calculation is more accurate to the CPU mode then disregard my comment!